package.json

```diff
@@ -19,7 +19,8 @@
         "utils:readmes": "node ./packages/dev-utils/build/index.js readmes",
         "utils:packages": "node ./packages/dev-utils/build/index.js packages",
         "utils:dependencies": "node ./packages/dev-utils/build/index.js dependencies && pnpm prettier:fix",
-        "check": "pnpm eslint:check && pnpm prettier:check && pnpm types:check"
+        "check": "pnpm eslint:check && pnpm prettier:check && pnpm types:check",
+        "postinstall": "node packages/testing-library/node_modules/puppeteer/install.mjs"
```

Comment:
This `postinstall` script should already exist in puppeteer's `package.json`. For some unknown reason, it's not executing in CI for us. Applying it here manually fixes that.

Reply:
I wonder if it's worth trying to use playwright instead since that's what we use the other SvelteJS projects? It looks like a relatively small number of places that puppeteer is referenced and I know we don't have to do this with Playwright. Though we do some custom stuff on the CI which I believe is to only download the relevant browser instead of all of them to speed things up: https://github.com/sveltejs/kit/blob/main/.github/workflows/ci.yml

Anyway, if we want to stick with puppeteer, I wonder if we should try setting the side-effects -cache to false instead as I think that's probably what's happening: https://pnpm.io/npmrc#side-effects-cache

Reply:
> I wonder if it's worth trying to use playwright instead since that's what we use the other SvelteJS projects? It looks like a relatively small number of places that puppeteer is referenced and I know we don't have to do this with Playwright.

I tried doing this a couple of weeks ago _before_ I had a good understanding of the codebase and failed magnificently 😄 Wasn't as clean of a swap-out that I thought it would be. I'm willing to give it another shot though

But yea, regarding tests, I'd really like to spend some time improving the testing suite as a whole. atm, it's a bit slow and the tests aren't truly executing in parallel (despite the use of the, deceptively named, [`promise-parallel-throttle`](https://github.com/svelte-add/svelte-add/blob/62db0c9be7f737dcd850968d6b936e671a84c935/packages/testing-library/package.json#L8) package)

> Anyway, if we want to stick with puppeteer, I wonder if we should try setting the side-effects -cache to false instead as I think that's probably what's happening: [pnpm.io/npmrc#side-effects-cache](https://pnpm.io/npmrc#side-effects-cache)

thanks for the suggestion! I'll see if this works